composer.json

```diff
@@ -14,9 +14,17 @@
     "friendsofphp/php-cs-fixer": "^2.5",
     "jakub-onderka/php-console-highlighter": "^0.3.2",
     "jakub-onderka/php-parallel-lint": "^0.9.2",
+    "localheinz/composer-normalize": "^0.1.0",
     "phake/phake": "^3.0.1",
     "phpunit/phpunit": "^6.2"
   },
+  "require-dev": {},
```

Comment:
Was thinking, maybe I should add a normalizer that removes empty hashes, at least on a root level!

Reply:
That would be very welcome 👍